merlin/systems/workflow/base.py

```diff
@@ -66,15 +66,18 @@ def __init__(self, workflow, output_dtypes, model_config, model_device):
             )
 
         # recurse over all column groups, initializing operators for inference pipeline.
-        # (disabled for now while we sort out whether and how we want to use C++ implementations
-        # of NVTabular operators for performance optimization)
-        # self._initialize_ops(self.workflow.output_node)
+        # (disabled everyting other than Categorify for now while we sort out whether
+        # and how we want to use C++ implementations of NVTabular operators for
+        # performance optimization)
+        self._initialize_ops(self.workflow.output_node, restrict=["Categorify"])
 
-    def _initialize_ops(self, workflow_node, visited=None):
+    def _initialize_ops(self, workflow_node, visited=None, restrict=False):
```

Comment:
perhaps could change this to `restrict=None` and set `restrict = restrict or []` in the method to keep the type consistent as an optional list of node names